### PR TITLE
Update README with instructions for AWSVPC mode

### PR DESCRIPTION
**Pull request by bharathrajbk**
### Summary

This pull request updates the documentation for launching ecs agent with the awsvpc networking mode use case on non Amazon linux AMIs
### Implementation details

This is only a readme change
### Testing
README update

### Description for the changelog

* Documentation update - Provide instructions for launching ecs-agent with awsvpc networking mode. 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,45 @@ $ docker run --name ecs-agent \
...
+--cap-add=net_admin \
+--env ECS_ENABLE_TASK_ENI=true \
+--env ENI_DHCLIENT_LEASES_PATH=/usr/sbin/dhclient \
+--env ECS_UPDATES_ENABLED=true \
+--env ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION=1h \
+--env ECS_ENABLE_CONTAINER_METADATA=false \
+--env ECS_DATADIR=/data \
+--env ECS_ENABLE_TASK_IAM_ROLE=true \
+--env ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST=true \
+--env ECS_LOFILE=/log/ecs-agent.log \
```
Comment (the0ffh) [inline README.md]:
```suggestion
--env ECS_LOGFILE=/log/ecs-agent.log \
```

Comment (bharathrajbk) [reply README.md]: @the0ffh , good find! Thank you. I have fixed this. Please review

File: README.md [hunk 2]
```diff
@@ -49,6 +49,45 @@ $ docker run --name ecs-agent \
...
+--env ECS_ENABLE_TASK_ENI=true \
+--env ENI_DHCLIENT_LEASES_PATH=/usr/sbin/dhclient \
+--env ECS_UPDATES_ENABLED=true \
+--env ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION=1h \
+--env ECS_ENABLE_CONTAINER_METADATA=false \
+--env ECS_DATADIR=/data \
+--env ECS_ENABLE_TASK_IAM_ROLE=true \
+--env ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST=true \
+--env ECS_LOGFILE=/log/ecs-agent.log \
+--env ECS_AVAILABLE_LOGGING_DRIVERS=["json-file","awslogs","splunk","syslog","none"] \
```
Comment (adnxn) [inline README.md]: `splunk` doesnt need to be included here

File: README.md [hunk 3]
```diff
@@ -49,6 +49,45 @@ $ docker run --name ecs-agent \
...
+--env ECS_LOGLEVEL=info \
+amazon/amazon-ecs-agent:latest
+```
```
Comment (adnxn) [inline README.md]: please add `-d`/`--detach` flag as well

**Comment by adnxn** [comment]
@bharathrajbk: okay great! please rebase to a single commit and descriptive commit message please.

**Comment by bharathrajbk** [comment]
@adnxn , this is done. Please review and merge

**Pull request merged**